tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Features/Order Update Status.feature

```diff
@@ -151,7 +151,7 @@ Scenario: When an order is complete, an the funding source is true, but the orde
     And all attachments use UTF8 encoding
     And the 'full' attachment contains the following data
         | Call Off Agreement ID | Call Off Ordering Party ID | Call Off Ordering Party Name | Call Off Commencement Date | Service Recipient ID | Service Recipient Name | Service Recipient Item ID | Supplier ID | Supplier Name | Product ID | Product Name | Product Type       | Quantity Ordered | Unit of Order | Unit Time | Estimation Period | Price   | Order Type | Funding Type | M1 planned (Delivery Date) | Actual M1 date | Buyer verification date (M2) | Cease Date |
-        | C000014-04            | OrgOds                     | NHS NORTHUMBERLAND CCG       | 07/05/2020                 | au                   | AU Test                | C000014-04-au-5           | 10101       | Supplier 1    | 100001-001 | Item 5       | Additional Service | 1                | per patient   | per month | per month         | 599.990 | 1          | Central      | 05/09/2021                 |                |                              |            |
+        | C000014-04            | OrgOds                     | NHS NORTHUMBERLAND CCG       | 07/05/2020                 | au                   | AU Test                | C000014-04-au-5           | 10101       | Supplier 1    | 100001-001 | Item 5       | Additional Service | 1                | per patient   | per month | per month         | 599.9900 | 1          | Central      | 05/09/2021                 |                |                              |            |

```

Comment:
I think it would make sense to use `599.9999` as the price here. I know the original was `599.990` but I think that might have been better as `599.999`. (This applies to all of the prices changed in this file.)

Can you also make sure that the vertical bars are all aligned? (This applies to all of the tables changed in this file.)

Reply:
I have updated data and also aligned table in feature file.